definitions/procedures/pulp/remove.rb

```diff
@@ -22,17 +22,29 @@ def pulp_data_dirs
       ]
     end
 
+    # rubocop:disable  Metrics/MethodLength
     def pulp_packages
-      [
-        'pulp-server', 'python-pulp-streamer', 'pulp-puppet-plugins',
-        'python-pulp-rpm-common', 'python-pulp-common',
-        'pulp-selinux', 'python-pulp-oid_validation',
-        'python-pulp-puppet-common', 'python-pulp-repoauth',
-        'pulp-rpm-plugins', 'python-blinker', 'python-celery',
-        'python-django', 'python-isodate', 'python-ldap',
-        'python-mongoengine', 'python-nectar', 'python-oauth2',
-        'python-pymongo'
-      ]
+      possible = %w[pulp-admin-client pulp-agent pulp-consumer-client pulp-deb-admin-extensions
+                    pulp-deb-plugins pulp-docker-admin-extensions pulp-docker-plugins
+                    pulp-nodes-admin-extensions pulp-nodes-child pulp-nodes-common
+                    pulp-nodes-consumer-extensions pulp-nodes-parent pulp-ostree-admin-extensions
+                    pulp-ostree-plugins pulp-puppet-admin-extensions
+                    pulp-puppet-consumer-extensions pulp-puppet-handlers pulp-puppet-plugins
+                    pulp-puppet-tools pulp-python-admin-extensions pulp-python-plugins
+                    pulp-rpm-admin-extensions pulp-rpm-consumer-extensions pulp-rpm-handlers
+                    pulp-rpm-plugins pulp-rpm-yumplugins pulp-selinux pulp-server python-bson
```

Comment:
Can't you use `rpm -qa | grep ^pulp-` or will that also match pulp 3 packages?

Reply:
I think both these suggestions would work, but to be honest i prefer this explicit list especially given the side effects we saw from users.